code/modules/power/tesla/energy_ball.dm

```diff
@@ -81,7 +84,7 @@
 
 	var/move_dir = 0
 	if(target && prob(75))
-		move_dir = get_dir(src,target)
+		move_dir = get_dir(src, target)
```

Comment:
How does this compile?

Reply:
Why wouldn't it?

Reply:
It doesn't care if you pass in loc or object.